packages/babel-generator/src/generators/classes.js

```diff
@@ -60,6 +60,7 @@ export function ClassProperty(node: Object) {
     this.print(node.key, node);
     this.token("]");
   } else {
+    this.print(node.variance, node);
```

Comment:
Because this has added this new `variance` property to all of these nodes, `variance` needs to be added to the `visitor` list in https://github.com/babel/babel/blob/master/packages/babel-types/src/definitions/flow.js#L46 and all of the other node types that now have a `.variance` property. Otherwise, `strip-flow-types` will miss them and not strip them.